apps/src/aichat/redux/selectors/index.ts

```diff
@@ -48,9 +48,14 @@ export const selectHavePropertiesChanged = (state: RootState) =>
     state.aichat.currentAiCustomizations
   ).length > 0;
 
-export const selectMultimodalEnabled = (state: RootState) => {
-  const isChatGpt =
-    state.aichat.currentAiCustomizations.selectedModelId ===
-    AiChatModelIds.CHATGPT;
-  return isChatGpt && queryParams('multimodal') === 'true';
+export const selectMultimodalEnabled = ({aichat, lab}: RootState) => {
+  const multimodalSupported = modelDescriptions.find(
```

Comment:
Is this sort of a double check in case we update a level and make multimodal not supported? This value being different than `multimodalEnabled` shouldn't generally be possible, right (ie, selected model is multimodal, but level doesn't have multimodal enabled)?

Reply:
Yep, but also in the future if we decide to support a combination of multimodal and non-multimodal models on one level, this will still work I believe. Since we check against the saved model, you'd have to commit the update before seeing multimodal features, and at that point you would've started a new chat context anyway.